src/js/game/GameController.js

```diff
@@ -219,13 +219,15 @@ class GameController {
         x: -450, alpha: 0.5
       }, this.timeout, Phaser.Easing.Linear.None);
 
+      tween.timeScale = 1;
       tween.start();
       tween = this.levelView.addResettableTween().to({
       }, this.timeout, Phaser.Easing.Linear.None);
 
       tween.onComplete.add(() => {
         this.endLevel(this.timeoutResult(this.levelModel));
       });
+      tween.timeScale = 1;
```

Comment:
Under integration tests, [the default `timeScale` is increased so tests run faster](https://github.com/code-dot-org/code-dot-org/pull/17114).

This tween is used to measure the level timeout... so I need to run it at normal speed.  In theory if all tweens are sped up by an equal amount they should still run in order, but it wasn't consistently passing.

I think when everything is run 100x there's some non-deterministic behavior of the timers (two callbacks may run in the same time slice, instead of sequentially).